plugins/modules/system/python_requirements_info.py

```diff
@@ -106,11 +115,28 @@
     '==': operator.eq,
 }
 
+if sys.version.startswith('2.6'):
+    python_version_info = dict(
+        major=sys.version_info[0],
+        minor=sys.version_info[1],
+        micro=sys.version_info[2],
+        releaselevel=sys.version_info[3],
+        serial=sys.version_info[4],
+    )
+else:
+    python_version_info = dict(
+        major=sys.version_info.major,
+        minor=sys.version_info.minor,
+        micro=sys.version_info.micro,
+        releaselevel=sys.version_info.releaselevel,
+        serial=sys.version_info.serial,
+    )
+
 
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            dependencies=dict(type='list', elements='str')
+            dependencies=dict(type='list', elements='str', default=[])
```

Comment:
I'm somewhat surprised that the sanity tests aren't screaming. The documentation does not say there is a default.

Reply:
I wonder if the sanity tests check are checking this in a boolean-ish way. Fixing docs now.

Reply:
fixed